locales/fr_FR/pcsx2_Main.po

```diff
@@ -52,11 +52,11 @@ msgstr ""
 
 #: common/src/Utilities/Exceptions.cpp:290
 msgid "Path: "
-msgstr "Chemin d'accès : "
+msgstr "Chemin : "
```

Comment:
Remove spaces before :

Reply:
French language has spaces before and after theses characters:  ? ! : ; % " + - / * =
See: https://www.blogdumoderateur.com/regles-ponctuation-typographie/